src/OpenLoco/src/Ui/Windows/ToolbarTop.cpp

```diff
@@ -990,49 +991,26 @@ namespace OpenLoco::Ui::Windows::ToolbarTop::Game
             window.widgets[Common::Widx::port_menu].image = Gfx::recolour(interface->img + InterfaceSkin::ImageIds::toolbar_ports);
         }
 
-        if (getGameState().lastRoadOption != 0xFF)
-        {
-            window.widgets[Common::Widx::road_menu].type = WidgetType::toolbarTab;
-        }
-        else
-        {
-            window.widgets[Common::Widx::road_menu].type = WidgetType::none;
-        }
-
-        if (getGameState().lastRailroadOption != 0xFF)
-        {
-            window.widgets[Common::Widx::railroad_menu].type = WidgetType::toolbarTab;
-        }
-        else
-        {
-            window.widgets[Common::Widx::railroad_menu].type = WidgetType::none;
-        }
-
-        if (getGameState().lastAirport != 0xFF || getGameState().lastShipPort != 0xFF)
-        {
-            window.widgets[Common::Widx::port_menu].type = WidgetType::toolbarTab;
-        }
-        else
-        {
-            window.widgets[Common::Widx::port_menu].type = WidgetType::none;
-        }
+        window.widgets[Common::Widx::road_menu].hidden = !(getGameState().lastRoadOption != 0xFF);
+        window.widgets[Common::Widx::railroad_menu].hidden = !(getGameState().lastRailroadOption != 0xFF);
+        window.widgets[Common::Widx::port_menu].hidden = !(getGameState().lastAirport != 0xFF || getGameState().lastShipPort != 0xFF);
```

Comment:
This feels a bit roundabout. Can we simplify this to, e.g.:
```suggestion
        window.widgets[Common::Widx::road_menu].hidden = getGameState().lastRoadOption != 0xFF;
        window.widgets[Common::Widx::railroad_menu].hidden = getGameState().lastRailroadOption == 0xFF;
        window.widgets[Common::Widx::port_menu].hidden = getGameState().lastAirport == 0xFF && getGameState().lastShipPort == 0xFF;
```